src/main/java/org/cookieandkakao/babting/domain/food/service/MeetingPreferenceService.java

```diff
@@ -16,31 +21,57 @@
 @Service
 public class MeetingPreferenceService {
     private final MemberMeetingRepository memberMeetingRepository;
-    private final PreferenceFoodRepository preferenceFoodRepository;
+    private final MeetingPreferenceFoodRepository meetingPreferenceFoodRepository;
+    private final MeetingNonPreferenceFoodRepository meetingNonPreferenceFoodRepository;
     private final NonPreferenceFoodRepository nonPreferenceFoodRepository;
+    private final FoodRepositoryService foodRepositoryService;
 
     public MeetingPreferenceService(MemberMeetingRepository memberMeetingRepository,
-                                    PreferenceFoodRepository preferenceFoodRepository,
-                                    NonPreferenceFoodRepository nonPreferenceFoodRepository) {
+                                    MeetingPreferenceFoodRepository meetingPreferenceFoodRepository,
+                                    MeetingNonPreferenceFoodRepository meetingNonPreferenceFoodRepository,
+                                    NonPreferenceFoodRepository nonPreferenceFoodRepository,
+                                    FoodRepositoryService foodRepositoryService
+    ) {
         this.memberMeetingRepository = memberMeetingRepository;
-        this.preferenceFoodRepository = preferenceFoodRepository;
+        this.meetingPreferenceFoodRepository = meetingPreferenceFoodRepository;
+        this.meetingNonPreferenceFoodRepository = meetingNonPreferenceFoodRepository;
         this.nonPreferenceFoodRepository = nonPreferenceFoodRepository;
+        this.foodRepositoryService = foodRepositoryService;
     }
 
-    public Set<Long> getRecommendedFoodsForMeeting(Long meetingId) {
-        // 특정 모임의 모든 멤버 조회
+    public List<FoodPreferenceGetResponse> getRecommendedFoodDetailsForMeeting(Long meetingId) {
+        Set<Long> recommendedFoodIds = getRecommendedFoodsForMeeting(meetingId);
+        List<Food> foods = foodRepositoryService.findFoodsByIds(recommendedFoodIds);
+        return foods.stream()
+                .map(food -> new FoodPreferenceGetResponse(
+                        food.getFoodId(),
+                        food.getFoodCategory().getName(),
+                        food.getName()))
```

Comment:
```suggestion
                                        .map(FoodPreferenceGetResponse::fromPreferenceFood)
```
여기도 위와 같이 작성하는 것 어떤가요??

Reply:
앗 리팩터링 하다가 놓친 부분이 있었네요. 감사합니다!